torch/csrc/Module.cpp

```diff
@@ -1219,6 +1215,8 @@ void initIttBindings(PyObject* module);
 } // namespace torch
 #endif
 
+PyMethodDef* MPSModule_methods();
```

Comment:
If you're not using a header for this, leave a comment where it will be added.

Reply:
I think a new header for the single line `PyMethodDef* MPSModule_methods();`  is overkill. 
So do you want me to put a TODO comment like this?
```
// TODO: add module method declarations to a common header file
PyMethodDef* MPSModule_methods();
```

Reply:
Moved it to a header, similar to what we do with other modules like [csrc/multiprocessing/init.h](https://github.com/pytorch/pytorch/blob/master/torch/csrc/multiprocessing/init.h)